tests/batch_processing/conftest.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import asyncio
+
+from prometheus_client.registry import REGISTRY, CollectorRegistry
+
+from mlserver.server import MLServer
+from mlserver.settings import Settings, ModelSettings
+
+from ..utils import RESTClient, get_available_ports
+
+
+
+import os
+import pytest
+from ..conftest import TESTDATA_PATH
+from ..rest.conftest import model_registry, rest_server, rest_app, rest_client
+
+
+@pytest.fixture()
+def single_input():
+    return os.path.join(TESTDATA_PATH, "batch_processing", "single.txt")
+
+
+@pytest.fixture
+def settings(settings: Settings) -> Settings:
+    http_port, grpc_port, metrics_port = get_available_ports(3)
+    settings.http_port = http_port
+    settings.grpc_port = grpc_port
+    settings.metrics_port = metrics_port
+
+    return settings
+
+
+@pytest.fixture
+async def mlserver(
```

Comment:
If this is duplicated from other folders within the root `tests/` package, feel free to move it to the base `conftest.py` BTW

Reply:
Opened separate issue